cmd/helm/testdata/output/lint-chart-with-bad-subcharts-with-subcharts.txt

```diff
@@ -5,7 +5,7 @@
 	error unpacking bad-subchart in chart-with-bad-subcharts: validation: chart.metadata.name is required
 
 ==> Linting testdata/testcharts/chart-with-bad-subcharts/charts/bad-subchart
-[ERROR] Chart.yaml: name is required
```

Comment:
Many users check for this error message. Removing it altogether could break those use cases.

Can you please check whether the chart name's length is equal to zero. If it is, return this error. If it exceeds 250 characters, return the other message.

Thanks.

Reply:
done, and added a test for the length of the charts name